pages/place/delete/index.js

```diff
@@ -44,6 +44,7 @@ module.exports = settings => {
 
   app.use('/confirm', (req, res, next) => {
     confirm({
+      page: 'place.delete',
```

Comment:
This is the bug fix, everything else is just tidying up issues I noticed while testing.